js/lerna.json

```diff
@@ -5,5 +5,5 @@
   "packages": [
     "./"
   ],
-  "version": "0.4.10"
```

Comment:
We have automate CI for bumping, so no need to bump manually. Can you revert this.

Reply:
Thank you! I had a suspicion that that was the case. I'll revert the version numbers in a push.

Reply:
Done.